services/weatherApi.service.go

```diff
@@ -0,0 +1,85 @@
+package services
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/gofiber/fiber/v2"
+	"net/http"
+	"project_weather/controllers"
+)
+
+const (
+	dailyUrl  = "http://api.weatherapi.com/v1/forecast.json?key=%s&q=%s&days=1&aqi=no&alerts=no"
```

Comment:
As far as I am able to see - the only diff between these 3 is the `days` value. Why not extract this one as a parameter too and have just a single constant instead ?